cmd/init.go

```diff
@@ -53,16 +56,16 @@ var initCmd = &cobra.Command{
 		copyAsset("app/main.go", filepath.Join(desktopCmdPath, "main.go"))
 		copyAsset("app/options.go", filepath.Join(desktopCmdPath, "options.go"))
 		copyAsset("app/icon.png", filepath.Join(desktopAssetsPath, "icon.png"))
-		copyAsset("app/gitignore", filepath.Join("desktop", ".gitignore"))
+		copyAsset("app/gitignore", filepath.Join(buildPath, ".gitignore"))
 
 		wd, err := os.Getwd()
 		if err != nil {
 			fmt.Printf("hover: Failed to get working dir: %v\n", err)
 			os.Exit(1)
 		}
 
-		cmdGoModInit := exec.Command(goBin, "mod", "init", projectPath+"/desktop")
-		cmdGoModInit.Dir = filepath.Join(wd, "desktop")
+		cmdGoModInit := exec.Command(goBin, "mod", "init", projectPath+buildPath)
```

Comment:
Not directly in the project, but I was looking at the PR changes.
I think a `/` is needed before `buildPath` right?

Reply:
yes, the "/" is needed.
Thanks!